arbitrum-docs/for-devs/dev-tools-and-resources/monitoring-tools-block-explorers.mdx

```diff
@@ -10,13 +10,13 @@ import KnowMoreToolsBox from '../partials/_know-more-tools-box-partial.md';
 
 <KnowMoreToolsBox />
 
-When building decentralized applications or other blockchain-based projects, it's crucial to have a clear understanding of your transactions and smart contracts. Monitoring tools and block explorers provide valuable insights into chain usage, enabling you to analyze information and gather crucial statistics. In this page, we'll introduce a range of powerful tools that can help you efficiently track, manage, and optimize your Arbitrum-based projects for a seamless development experience. 
+The following tools will help you inspect and monitor the transactions, smart contracts, and general chain usage associated with your decentralized apps (dApps):
 
 ### Arbiscan (Etherscan based chain explorer)
 
-[Arbiscan](https://arbiscan.io/) is a prominent blockchain explorer and analytics platform that allows users to access and analyze public data on the Arbitrum network, such as transactions, wallet addresses, and smart contracts. Created by the [Etherscan team](https://etherscan.io/), this centralized, self-contained platform aims to make Arbitrum blockchains more accessible to everyday users. Arbiscan helps users understand their interactions with the blockchain, identify suspicious behavior, and engage directly with smart contracts.
```

Comment:
IMO a simple table that says "use this if you need to ______" would be more than good enough; we don't want to assume the responsibility of selling tools or maintaining wikipedia-like content for them. 

Something to consider as a future optimization: consolidating and trimming these "overview pages" as much as possible, and then incorporating the tools within purposeful how-tos that demonstrate how (and why) to use them in specific situations.